pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/BrokerRegistryTest.java

```diff
@@ -0,0 +1,313 @@
+/*
```

Comment:
I assume this class does not need a review here, as it is added in a separate PR, https://github.com/apache/pulsar/pull/18810.